### PR TITLE
Add WP-CLI command to sync ACF groups automatically

### PR DESCRIPTION
**Pull request by gtenaschuk**
Closes #[169]

- Add a new plugin to create a wp-cli command. The command will synchronize ACF groups when there is a sync available.
- Add WP-CLI ACF sync command to the refresh script
There is a command example in the plugin code.

### CODE REVIEW

File: wp-content/plugins/tw-acf-sync-helper/tw-acf-sync-helper.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+		/**
+		 * @var \ACF_Admin_Field_Groups $field_groups_class
+		 */
+		$field_groups_class = acf_get_instance( 'ACF_Admin_Field_Groups' );
+
+		$field_groups_class->setup_sync();
+
+		// Disable "Local JSON" controller to prevent the .json file from being modified during import.
+		acf_update_setting( 'json', false );
```
Comment (rpeterman-gp) [inline wp-content/plugins/tw-acf-sync-helper/tw-acf-sync-helper.php:78]: Should we only need to do this if the settings is enable, and if so re-enable it after the import process is complete?

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]
Concerned that the script will disable the ACF JSON setting and not re-enable it, if the setting were previously enabled.

File: wp-content/plugins/tw-acf-sync-helper/tw-acf-sync-helper.php (hunk 1 above)
Comment (Sean12mps) [reply wp-content/plugins/tw-acf-sync-helper/tw-acf-sync-helper.php:78]: Hi Rick, 

the function is just a shorthand to adding a filter in ACF. I've also already made some tests. Changing a field group will still update the .json file for it.

This is a reference to how the function is being used.
https://www.advancedcustomfields.com/resources/acf-settings

If the function would update anything permanently, it'll be strange for ACF Team to recommend running it in their plugin init.

Comment (gtenaschuk) [reply wp-content/plugins/tw-acf-sync-helper/tw-acf-sync-helper.php:78]: @rpeterman-gp The acf_update_setting is used to change the acf settings on load time, it is not firing any database update. So, the json setting will be disabled just until the wp-cli  process ends.

**Review by rpeterman-gp — APPROVED** [review]

**Pull request merged**